addons/im_livechat/controllers/main.py

```diff
@@ -190,7 +190,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
                 'name': channel_vals['name'],
                 'chatbot_current_step_id': channel_vals['chatbot_current_step_id'],
                 'state': 'open',
-                'operator_pid': (operator_partner.id, display_name.replace(',', '')),
+                'operator': {'name': display_name, **operator_partner.mail_partner_format(fields={'id': True, 'write_date': True})[operator_partner]},
```

Comment:
still have an issue here, "name", should not be returned when it contains something else than name

Reply:
I think we should do a prep PR to clear the formatter here first, before we want to use/expand it for other things.